telegram/bot.py

```diff
@@ -1926,7 +1931,9 @@ def answer_shipping_query(self,
                               shipping_query_id,
                               ok,
                               shipping_options=None,
-                              error_message=None):
+                              error_message=None,
+                              timeout=None,
```

Comment:
timeout should not be explicit here. please remove it. (it should be part of kwargs)